Documentation/filters-strings.md

```diff
@@ -57,15 +57,17 @@ Removes all newlines and whitespace characters from the string.
 | `test \n\t `          | `test`                |
 | `test\n  test`        | `testtest`            |
 | `\r\ntest\n  test\n`  | `testtest`            |
+| ` test test `         | `test test`           |
 
-By default it removes whitespace characters, unless a single optional argument is set to "false", "no" or "0":
+By default it removes consecutive whitespace characters (and trims the ends), unless a single optional argument is set to "false", "no" or "0":
 
 | Input                 | Output                |
 |-----------------------|-----------------------|
 | `  \ntest`            | `  test`              |
 | `test \n\t `          | `test \t `            |
 | `test\n  test`        | `test  test`          |
 | `\r\ntest\n  test\n`  | `test  test`          |
+| ` test test `         | ` test test `         |
```

Comment:
Not sure why the beginning & end whitespaces would be kept there? I'd expect them to be removed by `.trimmingCharacters(in: .whitespacesAndNewlines)` here.